### PR TITLE
Fix setState() warning

### LINKED ISSUE #79: setState warning when Video is unmounted
Hi,
It seems that updateState is still called after Video component is unmounted. I think this is due to some events that are not correctly unbound.
The warning is :

> warning.js:36 Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the Video component.

I'm trying to fix this in a PR.

### PR DESCRIPTION
**Pull request by untemps**
Fix #79 
Call a function in componentWillUnmount to unbind events to updateState.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ node_modules
 npm-debug.log
 .DS_Store
 dist
+yarn.lock
```
Comment (tomkis1) [inline .gitignore]: You should commit the `yarn.lock` into the repo.

Comment (untemps) [inline .gitignore]: Are you sure ? It's specific to my environment.

Comment (tomkis1) [inline .gitignore]: http://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

although they claim if you are a lib author, then you should not commit the `yarn.lock` but the only reason for not committing the file is that it's redundant when you install the library. But still, it makes a huge sense for development.

**Comment by mderrick** [comment]
This should have been fixed in #79

**Pull request closed without merging**